libcbtest/callback.c

```diff
@@ -0,0 +1,55 @@
+#include <stdint.h>
```

Comment:
We need `//go:build ignore` for this file, as this file is unexpectedly included to compile the test.

Reply:
Never mind, the directory was separated.